v2/oauth/multiple-frontends-with-separate-backends.mdx

```diff
@@ -0,0 +1,286 @@
+      "grant_types": ["authorization_code", "refresh_token"],
+      "redirect_uri": ["https://<YOUR_APPLICATION_DOMAIN>/oauth/callback"]
+    }'
+```
+
+
+
+- `client_name` - A human-readable name of the client that will be used for identification.
+- `response_types` - Specifies the types of responses your client expects from the [**Authorization Server**](/docs/oauth/introduction#authorization-server). Most of the time, you would need the following two to be present:
+    - `code`: Indicates that the [**Client**](/docs/oauth/introduction#client) will receive an **Authorization Code** that will be exchanged for an [**OAuth2 Access Token**](/docs/oauth/introduction#oauth2-access-token).
+    - `id_token`: Indicates that the [**Client**](/docs/oauth/introduction#client) expects an [**ID Token**](/docs/oauth/introduction#id-token)
+- `grant_types` - The grant types that the [**Client**](/docs/oauth/introduction#client) will use. 
+    - `authorization_code`: Allows exchanging the **Authorization Code** for an [**OAuth2 Access Token**](/docs/oauth/introduction#oauth2-access-token).
+    - `refresh_token`: Allows exchanging the [**OAuth2 Refresh Token**](/docs/oauth/introduction#oauth2-refresh-token) for a new [**OAuth2 Access Token**](/docs/oauth/introduction#oauth2-access-token).
+- `redirect_uri` - A list of URIs to which the [**Authorization Server**](/docs/oauth/introduction#authorization-server) will send the user-agent (browser) after completing the authorization step. These can be deep links to mobile or desktop apps as well, but they must be exact URLs, without wildcards.
+
+If the creation was successful, the API will return a response that looks like this:
+
+```json
+
+{
+    "client_name": "<YOUR_CLIENT_NAME>",
+    "client_id": "<CLIENT_ID>",
+    "client_secret": "<CLIENT_SECRET>",
+    "callback_urls": ["https://<YOUR_APPLICATION_DOMAIN>/oauth/callback"],
+    "authorize_url": "^{form_apiDomain}/^{form_apiBasePath}/oauth/auth",
+    "token_fetch_url": "^{form_apiDomain}/^{form_apiBasePath}/oauth/token"
+}
+
+```
+
+:::caution
+
+You will have to save this response because we do not persist it interaly for security reasons.
+In the next steps we will use the values to complete several configurations.
+
+:::
+
+
+
+## 3. Set Up your Authorization Service 
+
+### Configure the Authorization Service Backend
+
+In your [**Authorization Service**](/docs/oauth/introduction#authorization-service) you will need to initialize the **OAuth2Provider** recipe.
+The recipe will expose the endpoints needed for enabling the [**OAuth 2.0**](/docs/oauth/introduction#oauth-20) flow.
+
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
+Update the `supertokens.init` call to include the `OAuth2Provider` recipe.
+
+1. Add the import statement for the recipe.
+
+```typescript 
+import OAuth2Provider from "supertokens-node/recipe/oauth2provider";
+```
+
+2. Update the list of recipes with the new initialization step.
+
+```tsx
+OAuth2Provider.init()
+```
```

Comment:
normally we show the full supertokens.init here and highlight the important parts. FOr some people it can get confusing where to do OAuth2Provider.init() if they do not really understand the recipe thing we have and just copy / paste docs blindly

Reply:
If we show the entire init page it might get confusing for them since we can't really know what recipes they already have initialized. Also if they don't understand where recipe init happens then we can't expect them to follow the OAuth2 session guides.